clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/ReservationReportCachingAgent.groovy

```diff
@@ -195,4 +229,26 @@ class ReservationReportCachingAgent implements CachingAgent {
         return AmazonReservationReport.OperatingSystemType.UNKNOWN
     }
   }
+
+  static class AccountReservationDetailSerializer extends JsonSerializer<AmazonReservationReport.AccountReservationDetail> {
+    ObjectMapper objectMapper = new ObjectMapper()
+    boolean mergeVpcReservations
+
+
+    @Override
+    void serialize(AmazonReservationReport.AccountReservationDetail value,
+                   JsonGenerator gen,
+                   SerializerProvider serializers) throws IOException, JsonProcessingException {
+      if (mergeVpcReservations) {
+        value = new AmazonReservationReport.AccountReservationDetail(
+          reserved: new AtomicInteger(value.reserved.intValue() + value.reservedVpc.intValue()),
+          used: new AtomicInteger(value.used.intValue() + value.usedVpc.intValue()),
+          reservedVpc: null,
```

Comment:
`null` values won't be serialized ... with `mergeVpcReservations == true` the generated report should be no different than it is right now.